### PR TITLE
refactor(api): do not home pipette plungers unless necessary

### PR DESCRIPTION
**Pull request by ahiuchingau**
# Overview

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

File: api/src/opentrons/hardware_control/ot3api.py [hunk 1]
```diff
@@ -1578,9 +1582,11 @@ async def pick_up_tip(
         # This extra shake ensures those tips are removed
         for rel_point, speed in spec.shake_off_list:
             await self.move_rel(realmount, rel_point, speed=speed)
-        # Here we add in the debounce distance for the switch as
-        # a safety precaution
-        await self.retract(realmount, spec.retract_target)
+
+        if not self._backend.check_motor_status([OT3Axis.by_mount(realmount)]):
```
Comment (sfoster1) [inline api/src/opentrons/hardware_control/ot3api.py:1586]: wait this should be part of https://github.com/Opentrons/opentrons/pull/12280 right? doesn't involve the plunger

Comment (sfoster1) [reply api/src/opentrons/hardware_control/ot3api.py:1586]: or really it conflicts, we remove the retract there I think

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Let's get this one a little more coordinated with https://github.com/Opentrons/opentrons/pull/12280 - it kind of seems like the only thing in here should be the `home_plunger` changes?

File: api/src/opentrons/hardware_control/ot3api.py (hunk 1 above)
Comment (Laura-Danielle) [reply api/src/opentrons/hardware_control/ot3api.py:1586]: yeah it should be removed here.

**Review by Laura-Danielle — COMMENTED** [review]
Hardware said they aren't comfortable having this behavior be the default right now :( They wanted to run some more tests.

**Comment by ahiuchingau** [comment]
Waiting for HW to confirm whether or not we want this behavior, closing the PR for now

**Pull request closed without merging**